src/lwc/gsAdminSetup/gsAdminSetup.html

```diff
@@ -0,0 +1,15 @@
+<template>
+    <div class="slds-card container">
+        <p class="slds-card__header-title">{title}</p>
+        <lightning-card>
```

Comment:
If I'm not mistaken, by using lightning-card you do need to wrap it in divs specifying the card container and title. See the [documentation](https://developer.salesforce.com/docs/component-library/bundle/lightning-card/documentation) for more examples.